packages/s/sunshine/abi_used_symbols

```diff
@@ -1,82 +1,19 @@
-UNKNOWN:_ZN5boost3log11v2_mt_posix5sinks26basic_text_ostream_backendIcE10auto_flushEb
-UNKNOWN:_ZN5boost3log11v2_mt_posix5sinks26basic_text_ostream_backendIcE5flushEv
-UNKNOWN:_ZN5boost3log11v2_mt_posix5sinks26basic_text_ostream_backendIcE7consumeERKNS1_11record_viewERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
-UNKNOWN:_ZN5boost3log11v2_mt_posix5sinks26basic_text_ostream_backendIcEC1Ev
-UNKNOWN:_ZN5boost3log11v2_mt_posix5sinks26basic_text_ostream_backendIcED1Ev
-UNKNOWN:_ZN5boost6locale4conv6to_utfIwEENSt7__cxx1112basic_stringIT_St11char_traitsIS5_ESaIS5_EEEPKcSB_RKNS4_IcS6_IcESaIcEEENS1_11method_typeE
-UNKNOWN:_ZTIN5boost10filesystem16filesystem_errorE
-UNKNOWN:_ZTIN5boost6detail16thread_data_baseE
-UNKNOWN:_ZTVN5boost6detail16thread_data_baseE
 ld-linux-x86-64.so.2:__tls_get_addr
-libX11.so.6:XKeysymToKeycode
 libayatana-appindicator3.so.1:app_indicator_get_type
 libayatana-appindicator3.so.1:app_indicator_new
-libayatana-appindicator3.so.1:app_indicator_set_icon
+libayatana-appindicator3.so.1:app_indicator_set_icon_full
 libayatana-appindicator3.so.1:app_indicator_set_menu
 libayatana-appindicator3.so.1:app_indicator_set_status
-libboost_filesystem.so.1.83.0:_ZN5boost10filesystem6detail15path_algorithms19find_root_directoryERKNS0_4pathE
-libboost_filesystem.so.1.83.0:_ZN5boost10filesystem6detail15path_algorithms21find_parent_path_sizeERKNS0_4pathE
-libboost_filesystem.so.1.83.0:_ZN5boost10filesystem6detail15path_algorithms9append_v3ERNS0_4pathEPKcS6_
-libboost_filesystem.so.1.83.0:_ZN5boost10filesystem6detail18create_directoriesERKNS0_4pathEPNS_6system10error_codeE
-libboost_filesystem.so.1.83.0:_ZN5boost10filesystem6detail6statusERKNS0_4pathEPNS_6system10error_codeE
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix11record_view11public_data7destroyEPKS3_
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix13attribute_set6insertENS1_14attribute_nameERKNS1_9attributeE
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix13attribute_setC1Ev
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix13attribute_setD1Ev
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix14attribute_name18get_id_from_stringEPKc
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix15unexpected_call6throw_EPKcmS4_
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux17code_convert_implEPKwmRNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEmRKSt6locale
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux17once_block_sentry6commitEv
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux17once_block_sentry8rollbackEv
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux18atomic_based_event13set_signalledEv
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux18atomic_based_event4waitEv
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux21threadsafe_queue_impl12unsafe_emptyEPKS3_
-libboost_log.so.1.83.0:_ZN5boost3log11v2_mt_posix3aux21threadsafe_queue_impl15reset_last_nodeEPS3_
```

Comment:
libboost couldn't be used during the build as the project requires a newer version (1.86.x, current in unstable is 1.83.0). 

The projects build with a fallback nonetheless, but I don't know if there any disadvantages to this?